Marlin/pins_MELZI_CREALITY.h

```diff
@@ -0,0 +1,109 @@
+/**
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * Melzi pin assignments / Creality Melzi variant
+ */
+
+#define BOARD_NAME "Melzi Creality"
+
+#ifdef __AVR_ATmega1284P__
+  #define LARGE_FLASH true
+#endif
+
+// For the stock CR-10 use the REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER
+//   option for the display in Configuration.h
+
+#define SANGUINOLOLU_V_1_2
+#include "pins_SANGUINOLOLU_11.h"
+
+#undef LCD_PINS_RS
+#undef LCD_PINS_ENABLE
+#undef LCD_PINS_D4
+#define LCD_PINS_RS 28		// st9720 CS
+#define LCD_PINS_ENABLE 17	// st9720 DAT
+#define LCD_PINS_D4 30		// st9720 CLK
+
+//Don't need these
+#undef LCD_SDSS
+#undef LED_PIN
+
+// Don't have these data lines for the Creality version of the display
+#ifdef LCD_PINS_D5
+#undef LCD_PINS_D5
+#define LCD_PINS_D5 -1
+#endif
```

Comment:
These 4 lines can simply be replaced by `#undef LCD_PINS_D5`, and the same goes for other similar instances.